src/views/List.jsx

```diff
@@ -23,18 +25,38 @@ export function List({ data }) {
 		setSearchTerm('');
 	};
 
+	const renderedListLength = renderedList.length;
+
 	return (
 		<>
-			<form onSubmit={clearSearchField}>
-				<label htmlFor="search-filter">Item name:</label>
-				<input
-					type="text"
-					id="search-filter"
-					value={searchTerm}
-					onChange={(e) => setSearchTerm(e.target.value)}
-				/>
-				<button type="submit"> clear </button>
-			</form>
+			{renderedListLength === 0 && (
+				<>
+					<h2>Your list currently has no items. </h2>
+					<h3>Click on the add first item button to start your list.</h3>
+					<Link to="/add-item">
+						<button> add first item</button>
+					</Link>
+				</>
+			)}
+			{renderedListLength > 0 && (
+				<>
+					<form onSubmit={clearSearchField}>
+						<label htmlFor="search-filter">
+							Search for an item in your list:
+						</label>
```

Comment:
It's usually a good idea to make our labels as short as possible. This saves space in our UI, eliminating a problem that Amy pointed out in her screenshot, and reduces clutter for users who are listening to our page via screen reader. Why not "Search" here?

Generally, you will find that labels for text inputs, radios, and the like _can_ be longer if they have to, but button labels are very short. I once worked on a team where buttons could only contain two words or fewer, for example.

Reply:
thank you EJ for this! This is good to know! I will update the button to "search"